src/game/music.c

```diff
@@ -82,6 +82,12 @@ bool Music_Play(int16_t track)
         return Sound_Effect(SFX_SECRET, NULL, SPM_ALWAYS);
     }
 
+    if (g_Config.fix_speeches_killing_music && track >= MX_BALDY_SPEECH
+        && track <= MX_SKATEKID_SPEECH) {
+        return Sound_Effect(
+            SFX_BALDY_SPEECH + track - MX_BALDY_SPEECH, NULL, SPM_ALWAYS);
+    }
```

Comment:
Sanity check: won't this cause an audible degradation of sound fidelity?

Reply:
Yes the secret killing music fix also has the problem. Is there a way for the audio handler to play music types (more a question for rr probably)?

Reply:
To my ears this sounds almost indistinguishable, I really have to look out for the difference, even on headphones.
Please note that I'm hard of hearing so I'm not the best person to judge this.